<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanMakeConclusionForm</class>
 <widget class="QWidget" name="CanMakeConclusionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>210</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>210</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="conclusionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>58</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>78</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225)</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Можете ли Вы сделать заключение?</string>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Вопрос:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="answerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualiseButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Визуализировать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
